support-frontend/assets/helpers/user/user.ts

```diff
@@ -49,7 +50,9 @@ function getUserStateField(): string | undefined {
 const isPostDeployUser = (): boolean =>
 	cookie.get('_post_deploy_user') === 'true';
 
+export const REDIRECT_AFTER_SIGNOUT_STORAGE_KEY = 'redirectAfterSignOut';
```

Comment:
Not entirely sure this module is the best place to have this logic yet, we want to run this as early as possible in the page's javascript, so we might need to find the entry point for the Newspaper Checkout landing page. 🤔

Reply:
@LAKSHMIRPILLAI the entry point for the Paper checkout is`support-frontend/assets/pages/paper-subscription-checkout/paperSubscriptionCheckout.tsx` I think this would be the most appropriate place for this logic.

Reply:
@GHaberis I tried to move REDIRECT_AFTER_SIGNOUT_STORAGE_KEY to paperSubscriptionCheckout.tsx , but it seems to raise some cyclical dependencies.So I think, it's best to keep it here for now